src/components/sidebar/sidebar.less

```diff
@@ -125,21 +125,25 @@
     display: block;
```

Comment:
wouldn't display: flex work better here?

Reply:
ok i tried with flexbox, and it doesn't really budge. i guess the html is really off here, eh

Reply:
I also tried flex, it gets messy because of the hover effect that expands the button.